ee/session_recordings/ai/error_clustering.py

```diff
@@ -0,0 +1,61 @@
+from prometheus_client import Histogram
+from django.conf import settings
+from posthog.clickhouse.client import sync_execute
+from posthog.models.team import Team
+from sklearn.cluster import DBSCAN
+import pandas as pd
+
+CLUSTER_REPLAY_ERRORS_TIMING = Histogram(
+    "posthog_session_recordings_cluster_replay_errors",
+    "Time spent clustering the embeddings of replay errors",
+)
```

Comment:
should we have specific buckets here? is this likely slower than an API call?

Reply:
Yep, good catch. Much more so - I'd expect the clustering to be measured in seconds